puppet/modules/quickstack/manifests/neutron/controller.pp

```diff
@@ -326,10 +325,8 @@
       vxlan_group           => $ml2_vxlan_group,
       vni_ranges            => $ml2_vni_ranges,
       enable_security_group => str2bool_i("$ml2_security_group"),
-      firewall_driver       => $ml2_firewall_driver,
-    }
 
-    # If cisco nexus is part of ml2 mechanism drivers,
+    # If cesco nexus is part of ml2 mechanism drivers,
```

Comment:
Bump, if you have a chance to fix this, I'd like to merge, would rather not mess with your commit, or add another just to fix a typo